### PR TITLE
require sphinx version 1.5.1

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

File: doc/checkvers.py [hunk 1]
```diff
@@ -28,9 +28,9 @@
   # Check whether we have a recent version of sphinx. EPEL and CentOS are completely crazy and I don't understand their
   # packaging at all. The test below works on Ubuntu and places where sphinx is installed sanely AFAICT.
   if options.checkvers:
-    print 'checking for sphinx version >= 1.2... ',
-    # Need at least 1.2 because of some command line options stuff HRP added.
-    # Also 1.2 guarantees sphinx.version_info is available.
+    print 'checking for sphinx version >= 1.5.1... ',
+    # Need at least 1.5.1 to use svg
+    # Also 1.5.1 guarantees sphinx.version_info is available.
```
Comment (SolidWallOfCode) [inline doc/checkvers.py]: I think you made this comment inaccurate.

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/101/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/115/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/98/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1797/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1689/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/227/

File: doc/checkvers.py (hunk 1 above)
Comment (persiaAziz) [inline doc/checkvers.py]: Fixed

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/116/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/102/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/359/

**Comment by SolidWallOfCode** [comment]
Thanks, @persiaAziz.
This closes #1497 - require a sufficient version of Sphinx Docs.

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/99/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1798/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1690/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/228/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/360/

**Comment by zwoop** [comment]
Why was this merged without setting appropriate labels / versions?

**Pull request merged**